android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/data/network/model/ApiMovieDetails.kt

```diff
@@ -0,0 +1,38 @@
+package agency.five.codebase.android.movieapp.data.network.model
+
+import agency.five.codebase.android.movieapp.data.network.BASE_IMAGE_URL
+import agency.five.codebase.android.movieapp.model.Movie
+import agency.five.codebase.android.movieapp.model.MovieDetails
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ApiMovieDetails(
+    @SerialName("id") val id: Int,
+    @SerialName("original_language") val original_language: String,
+    @SerialName("overview") val overview: String?,
+    @SerialName("poster_path") val poster_path: String?,
+    @SerialName("release_date") val release_date: String,
+    @SerialName("runtime") val runtime: Int?,
+    @SerialName("title") val title: String,
+    @SerialName("vote_average") val vote_average: Double,
```

Comment:
You should have consistent naming convention. vote_average -> voteAverage.  same for the rest.